submissions/oluwaboriShobayo/HTML/productPage.html

```diff
@@ -11,6 +11,8 @@ <h1><strong>STYLE BY BORI</strong></h1>
             <li><a href="#">Home</a></li>
             <li><a href="#">Categories</a></li>
             <li><a href="#">Cart</a></li>
+            <li><a href="html-form/contactForm.html">Contact Us</a></li>
```

Comment:
Are you sure this link redirects to the contact form page? Your contactForm.html file is in "html-forms" folder. I can see you are using "html-form" here, without the s. Kindly confirm that the links navigate correctly.